helix-core/src/main/java/org/apache/helix/controller/rebalancer/topology/Topology.java

```diff
@@ -204,8 +172,9 @@ private Node createClusterTree() {
       InstanceConfig insConfig = _instanceConfigMap.get(instanceName);
       try {
         LinkedHashMap<String, String> instanceTopologyMap =
-            computeInstanceTopologyMap(_clusterConfig.isTopologyAwareEnabled(), instanceName,
-                insConfig, _clusterTopologyKeys);
+            computeInstanceTopologyMapHelper(_clusterConfig.isTopologyAwareEnabled(), instanceName,
```

Comment:
I believe you meant to call `computeInstanceTopologyMap()` with a false in the end here, instead of the concrete function itself?

Reply:
TFTR. I think computeInstanceTopologyMap for users outside this function. They need to call 'computeInstanceTopologyMap' to construct the 'clusterTopologyConfig' (line 323) first and then build the instance topology path. Here in Topology it self, the 'clusterTopologyConfig' is computed in the constructor. So we need another layer of abstraction. 
I am open to discussion.

Reply:
I see. Typically if a concrete function involves a null value, such as this case, it usually isn't called directly by outside callers and is only called by its wrapper. I'm not 100% certain with the convention here so I'll leave it to you to decide. Thank you for explaining!